addons/rook/template/testgrid/k8s-docker.yaml

```diff
@@ -0,0 +1,9 @@
+kubernetes:
+  version: "latest"
+weave:
+  version: "latest"
+docker:
+  version: "latest"
+rook:
+  version: "__testver__"
+  s3Override: "__testdist__"
```

Comment:
IMO we should also include minio, kotsadm or registry here to use a PVC and ensure rook is working

Reply:
adding minio